git-setup.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+# If no project name is given
```

Comment:
Possible to use https://github.com/google/zx instead of shell script? 
Makes it easy to understand and contribute in it.